diff --git a/cjs/extractLiveQueryRootFieldCoordinates.js b/cjs/extractLiveQueryRootFieldCoordinates.js
index 5b04d50b57a5ad66f815dd0713c0c9958f62c5bd..cdfd9ae48f1aa83e77e26e71b645916ed9cd498f 100644
--- a/cjs/extractLiveQueryRootFieldCoordinates.js
+++ b/cjs/extractLiveQueryRootFieldCoordinates.js
@@ -43,7 +43,7 @@ const extractLiveQueryRootFieldCoordinates = (params) => {
             const parentType = params.typeInfo.getParentType();
             if ((0, Maybe_js_1.isSome)(parentType) &&
                 parentType.name === "Query" &&
-                (0, Maybe_js_1.isSome)((_a = fieldNode.arguments) === null || _a === void 0 ? void 0 : _a.length)) {
+                (0, Maybe_js_1.isSome)(((_a = fieldNode.arguments) !== null && _a !== void 0 ? _a : []).length)) {
                 const fieldDef = params.typeInfo.getFieldDef();
                 identifier.add(`Query.${fieldNode.name.value}`);
                 if ((0, Maybe_js_1.isSome)(fieldDef)) {
diff --git a/esm/extractLiveQueryRootFieldCoordinates.js b/esm/extractLiveQueryRootFieldCoordinates.js
index 8f0d743e73a035bc01a94325886f0e85681d4c48..5be7efd97d4f514951893d3018f9453673d9f76b 100644
--- a/esm/extractLiveQueryRootFieldCoordinates.js
+++ b/esm/extractLiveQueryRootFieldCoordinates.js
@@ -40,7 +40,7 @@ export const extractLiveQueryRootFieldCoordinates = (params) => {
             const parentType = params.typeInfo.getParentType();
             if (isSome(parentType) &&
                 parentType.name === "Query" &&
-                isSome((_a = fieldNode.arguments) === null || _a === void 0 ? void 0 : _a.length)) {
+                isSome(((_a = fieldNode.arguments) !== null && _a !== void 0 ? _a : []).length)) {
                 const fieldDef = params.typeInfo.getFieldDef();
                 identifier.add(`Query.${fieldNode.name.value}`);
                 if (isSome(fieldDef)) {
