diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index c508c63bec12c6065d0debdc16b0340bbbf11d5b..8a809503e79401b7eb241e4f300cff00dd032061 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -1113,4 +1113,6 @@ exports.minimatch.AST = ast_js_1.AST;
 exports.minimatch.Minimatch = Minimatch;
 exports.minimatch.escape = escape_js_1.escape;
 exports.minimatch.unescape = unescape_js_1.unescape;
+// Provide an ES-module-style default export for interop, aliasing the main `minimatch` export.
+exports.default = exports.minimatch;
 //# sourceMappingURL=index.js.map
