diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index c508c63bec12c6065d0debdc16b0340bbbf11d5b..2edd90ff5b4db9eee104840ef738d19bd19edb28 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -1113,4 +1113,5 @@ exports.minimatch.AST = ast_js_1.AST;
 exports.minimatch.Minimatch = Minimatch;
 exports.minimatch.escape = escape_js_1.escape;
 exports.minimatch.unescape = unescape_js_1.unescape;
+exports.default = exports.minimatch;
 //# sourceMappingURL=index.js.map
