diff --git a/index.js b/index.js
index 7b04a602b029e8457a56343b99f9f0ea8e8f0490..e2d388166713ac690ce9c56246bdd8b63dddf85c 100644
--- a/index.js
+++ b/index.js
@@ -1,10 +1,9 @@
-import process from 'node:process';
-import {findUp, findUpSync} from 'find-up-simple';
-
-export async function packageUp({cwd = process.cwd()} = {}) {
+module.exports.packageUp = async function packageUp({cwd = process.cwd()} = {}) {
+	const { findUp } = await import('find-up-simple');
 	return findUp('package.json', {cwd});
 }
 
-export function packageUpSync({cwd = process.cwd()} = {}) {
+module.exports.packageUpSync = function packageUpSync({cwd = process.cwd()} = {}) {
+	const { findUpSync } = require('find-up-simple');
 	return findUpSync('package.json', {cwd});
 }
